docs/guide/signing-data.md

```diff
@@ -197,16 +197,16 @@ signTypedDataV4Button.addEventListener('click', function (event) {
     },
   });
 
-  var from = web3.eth.accounts[0];
+  var from = web3.eth.getAccounts();
```

Comment:
Hi @matt5346 👋  Thanks for the update here. One thing, `getAccounts()` returns a promise so we'll need to wait for the promise to resolve before we can get `from[0]`